code/modules/atmospherics/machinery/binary/passive_gate.dm

```diff
@@ -59,9 +59,8 @@ obj/machinery/atmospherics/binary/passive_gate
 			if(network2)
 				network2.update = 1
 
-obj/machinery/atmospherics/binary/passive_gate/attackby(obj/weapon as obj, mob/user as mob)
-	..()
```

Comment:
Why get rid of the parent call?

Reply:
that way it doesn't show the "elan oots hits the ___ pump with a multitool!"

Reply:
wait I should do the parent call in the else